Packs/Reco/Integrations/Reco/Reco.py

```diff
@@ -0,0 +1,319 @@
+import base64
+from datetime import datetime
+import dateutil.parser
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
```

Comment:
It's considered bad practice to `import *`. Import the package and use the full name for things. Otherwise you risk unexpected domain pollution